packages/wrangler/src/event-subscriptions/products.ts

```diff
@@ -0,0 +1,74 @@
+import type { HandlerContext } from "../core/types";
+
+// Use the same `id` here as the one you used in queue-broker-worker
+export const PRODUCTS: ProductSpec[] = [
+	{
+		id: "clipboards",
```

Comment:
This is an entirely made-up product that I've been using to test things out before real teams onboard. I will remove this once the first team onboards -- we do not want to expose this to users because it is not real.